### ISSUE TITLE
Error 500 when starring or archiving an article with the wallabag app for Android

### ISSUE THREAD

**Issue by ddeimeke** [wallabag/wallabag#1946]
### Issue details

I use the recent version of the Wallabag app (Google Playstore). Reading an article and synchroizing works perfectly well. Marking article as archived or delete or favorite gives an error 500.

### Environment

* wallabag version (or git revision) that exhibits the issue: 2.0.1
* How did you install wallabag? Via `git clone` or by downloading the package? git clone
* Last wallabag version that did not exhibit the issue (if applicable): 1.9.1 (never tested it with any other 2.0.x version)
* php version: 5.6.20
* OS: CentOS 7.2 (recent patchlevel)
* type of hosting (shared or dedicated): dedicated
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): MariaDB

### Steps to reproduce/test case

Please provide necessary steps for reproduction of this issue, or better the
reduced test case (without any external dependencies, if possible).

```
[2016-04-19 06:58:19] request.INFO: Matched route "star_entry". {"route_parameters":{"_controller":"Wallabag\\CoreBundle\\Controller\\EntryController::toggleStarAction","id":"177","_route":"star_entry"},"request_uri":"https://wallabag.deimeke.net/star/177"} []
[2016-04-19 06:58:19] security.DEBUG: Read existing security token from the session. {"key":"_security_secured_area"} []
[2016-04-19 06:58:19] security.DEBUG: User was reloaded from a user provider. {"username":"dirk","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2016-04-19 06:58:19] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Cannot redirect to an empty URL." at /srv/www/wallabag.deimeke.net/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/RedirectResponse.php line 76 {"exception":"[object] (InvalidArgumentException(code: 0): Cannot redirect to an empty URL. at /srv/www/wallabag.deimeke.net/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/RedirectResponse.php:76)"} []
[2016-04-19 06:58:19] security.DEBUG: Stored the security token in the session. {"key":"_security_secured_area"} []
[2016-04-19 06:58:20] request.INFO: Matched route "star_entry". {"route_parameters":{"_controller":"Wallabag\\CoreBundle\\Controller\\EntryController::toggleStarAction","id":"177","_route":"star_entry"},"request_uri":"https://wallabag.deimeke.net/star/177"} []
[2016-04-19 06:58:20] security.DEBUG: Read existing security token from the session. {"key":"_security_secured_area"} []
[2016-04-19 06:58:20] security.DEBUG: User was reloaded from a user provider. {"username":"dirk","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2016-04-19 06:58:20] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Cannot redirect to an empty URL." at /srv/www/wallabag.deimeke.net/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/RedirectResponse.php line 76 {"exception":"[object] (InvalidArgumentException(code: 0): Cannot redirect to an empty URL. at /srv/www/wallabag.deimeke.net/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/RedirectResponse.php:76)"} []
[2016-04-19 06:58:20] security.DEBUG: Stored the security token in the session. {"key":"_security_secured_area"} []
[2016-04-19 06:58:25] request.INFO: Matched route "star_entry". {"route_parameters":{"_controller":"Wallabag\\CoreBundle\\Controller\\EntryController::toggleStarAction","id":"177","_route":"star_entry"},"request_uri":"https://wallabag.deimeke.net/star/177"} []
[2016-04-19 06:58:25] security.DEBUG: Read existing security token from the session. {"key":"_security_secured_area"} []
[2016-04-19 06:58:25] security.DEBUG: User was reloaded from a user provider. {"username":"dirk","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2016-04-19 06:58:25] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Cannot redirect to an empty URL." at /srv/www/wallabag.deimeke.net/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/RedirectResponse.php line 76 {"exception":"[object] (InvalidArgumentException(code: 0): Cannot redirect to an empty URL. at /srv/www/wallabag.deimeke.net/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/RedirectResponse.php:76)"} []
[2016-04-19 06:58:25] security.DEBUG: Stored the security token in the session. {"key":"_security_secured_area"} []
[2016-04-19 06:58:29] request.INFO: Matched route "archive_entry". {"route_parameters":{"_controller":"Wallabag\\CoreBundle\\Controller\\EntryController::toggleArchiveAction","id":"177","_route":"archive_entry"},"request_uri":"https://wallabag.deimeke.net/archive/177"} []
[2016-04-19 06:58:29] security.DEBUG: Read existing security token from the session. {"key":"_security_secured_area"} []
[2016-04-19 06:58:29] security.DEBUG: User was reloaded from a user provider. {"username":"dirk","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2016-04-19 06:58:29] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Cannot redirect to an empty URL." at /srv/www/wallabag.deimeke.net/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/RedirectResponse.php line 76 {"exception":"[object] (InvalidArgumentException(code: 0): Cannot redirect to an empty URL. at /srv/www/wallabag.deimeke.net/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/RedirectResponse.php:76)"} []
[2016-04-19 06:58:29] security.DEBUG: Stored the security token in the session. {"key":"_security_secured_area"} []
[2016-04-19 07:00:08] request.INFO: Matched route "delete_entry". {"route_parameters":{"_controller":"Wallabag\\CoreBundle\\Controller\\EntryController::deleteEntryAction","id":"178","_route":"delete_entry"},"request_uri":"https://wallabag.deimeke.net/delete/178"} []
[2016-04-19 07:00:08] security.DEBUG: Read existing security token from the session. {"key":"_security_secured_area"} []
[2016-04-19 07:00:08] security.DEBUG: User was reloaded from a user provider. {"username":"dirk","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2016-04-19 07:00:08] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Cannot redirect to an empty URL." at /srv/www/wallabag.deimeke.net/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/RedirectResponse.php line 76 {"exception":"[object] (InvalidArgumentException(code: 0): Cannot redirect to an empty URL. at /srv/www/wallabag.deimeke.net/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/RedirectResponse.php:76)"} []
[2016-04-19 07:00:08] security.DEBUG: Stored the security token in the session. {"key":"_security_secured_area"} []
```

**Comment by nicosomb** [comment]
Thank you for this detailed issue :+1: 

This bug is already fixed in #1925 ;-)

**Comment by ddeimeke** [comment]
Perfect, thank you!
